force-app/main/default/lwc/util/util.js

```diff
@@ -10,6 +10,7 @@
 import { ShowToastEvent } from "lightning/platformShowToastEvent";
 import unknownErrorLabel from "@salesforce/label/c.Util_Unknown_Error";
 import errorLabel from "@salesforce/label/c.Util_Error";
+import ISCLIENT_FIELD from "@salesforce/schema/Contact.IsClient__c";
```

Comment:
Do we need these changes on util anymore?

Reply:
We do not for this one. I thought we might keep it since we could use it sometime in pmdm